### PR TITLE
Automated cherry pick of #52: Update base image tag file(s)

### PR DESCRIPTION
**Pull request by jaxesn**
Cherry pick of #52 on release-0.1.

#52: Update base image tag file(s)

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note

```

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/61#" title="Author self-approved">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/release-0.1/OWNERS)~~ [jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/61#" title="Author self-approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/61#pullrequestreview-767016318" title="LGTM">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/release-0.1/OWNERS)~~ [TerryHowe,jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
